packages/models-library/src/models_library/api_schemas_webserver/product.py

```diff
@@ -0,0 +1,31 @@
+from decimal import Decimal
+from typing import Any, ClassVar
+
+from pydantic import Field, validator
+
+from ._base import OutputSchema
+
+
+class ProductPriceGet(OutputSchema):
+    product_name: str
+    usd_per_credit: Decimal | None = Field(
+        ...,
+        description="Price of a credit in USD. "
+        "If None, then this product's price is UNDEFINED",
+    )
+
+    @validator("usd_per_credit")
+    @classmethod
+    def non_negative(cls, v):
+        if v < 0:
+            msg = "Must be non-negative value, got {v}"
+            raise ValueError(msg)
+        return v
```

Comment:
Q: why would this in a get request be less than 0?
I would expect that the value in the DB should be already validated and valid, meaning > 0

Reply:
Since I see that this is also constraint in the db, I think this validator cam be omitted.

Reply:
Why this is coming from the db? :-) this is in principle part of the API and can be filled by any entity... 
I think the validation does not harm